Gemfile.lock

```diff
@@ -0,0 +1,158 @@
+GIT
```

Comment:
+1 for committing the .lock file. It helps backtrack the exact version number if you get in to gem version issues.